oeps/processes/oep-0055/decisions/0003-readme-specification.rst

```diff
@@ -0,0 +1,183 @@
+0003 README specification
+#########################
+
+Status
+******
+
+** Accepted **
+
+Context
+*******
+
+`OEP-55`_ states that one of the jobs of a maintainer is te maintain an
+up-to-date and useful readme. While it laid out some possibilites about what
+should go into a README, it did not go into much detail.
+
+Decision
+********
+
+We will define a specification that READMEs in the open edx project must
+follow.  By having a single definition, we can make it easier for people to
+navigate between projects and find the most important information quickly.
+
+Specification
+*************
+
+The readme of a well-maintained repo should contain at least the following
+sections.  Ideally in this exact order.
+
+* **Title** - The name of the component.
+
+* **Badges** - A set of `badges <https://github.com/badges/shields>`_ that
+  quickly let visitors review the health of the project.
+
+	 * The badges section should contain at minimum the following badges:
+
+		  * **License** - Indicating the license of the repo.
+
+		  * **Maintenance Status** - One of ``maintained``, ``deprecated``, ``unsupported``, ``experimental``.
+
+		  * **CI Status** - The status of CI testing on the main branch.
+
+		  * **Package Published Status** - If the component is a library, then a badge indicating whether publishes to the relevant package index are passing. Eg. Has the latest version been successfully published to PyPI or NPM?
+
+
+* **Purpose** - A paragraph or two summarizing the purpose of this component.
+
+* **Getting Started** - A section to indicate how you can begin development on
+  the new component.
+
+* **Getting Help** - A section to indicate where and how the user can get help
+  with the component.  It should link to relevant docs as well as places where
+  the users can get help with from other humans.
+
```

Comment:
Do we need a section about how to report issues?

Reply:
Other sections we might need:
- Supported platforms (Aspen--Nutmeg, Python 3.x?, OS?)
- Changelog or link to one?
- Status in Open edX: core, supported, optional, etc?
- Installation: where are the docs for how to start running this component?

Reply:
I'm not sure what will happen with this comment?